tutorials/export/exporting_for_web.rst

```diff
@@ -187,14 +213,6 @@ to remap them based on model/vendor/OS due to privacy considerations.
 
 .. warning:: Requires a :ref:`secure context <doc_javascript_secure_contexts>`.
 
-Boot splash is not displayed
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The default HTML page does not display the boot splash while loading. However,
-the image is exported as a PNG file, so :ref:`custom HTML pages <doc_customizing_html5_shell>`
-can display it.
-
-.. _doc_exporting_for_web_serving_the_files:
```

Comment:
This being deleted is causing the current build error. It's used to link from the bottom of the optimizing for size page [here](https://docs.godotengine.org/en/stable/contributing/development/compiling/optimizing_for_size.html).